front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/InputsTableRow.tsx

```diff
@@ -61,13 +68,32 @@ const InputsTableRow = ({ input, onSelect }: InputsTableRowProps) => {
       </td>
       <td>
         <CategoryList>
-          {input.relationships?.categories.data.map((category) => (
-            <Category id={category.id} inputId={input.id} key={category.id} />
-          ))}
+          {input.relationships?.categories.data
+            .filter((category) =>
+              query.category ? category.id === query.category : category
+            )
+            .map((category) => (
+              <Category id={category.id} inputId={input.id} key={category.id} />
+            ))}
         </CategoryList>
       </td>
+      {query.category && (
```

Comment:
That might be nitpicking a bit but I think this column is a bit far from the previous one, would put them closer as the designs do.

Reply:
Yes, indeed! Thanks for noticing it :)